nservicebus/azure-service-bus/error-handling.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Azure Service Bus Error Handling
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Native Error Handling
```

Comment:
I see what you mean now, the top header isn't necessary... here I also did it wrong http://docs.particular.net/nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees (But this one shows the comment I made, because of title length I dropped the Azure Service Bus part, now it doesn't look good here even if there were only 1 header)